web/src/pages/access/request/[id].tsx

```diff
@@ -512,28 +514,32 @@ const AccessRequestForm = () => {
                           max: rule?.timeConstraints.maxDurationSeconds,
                           min: 60,
                         }}
-                        render={({ field: { ref, ...rest } }) => {
-                          return (
-                            <DurationInput
-                              {...rest}
-                              max={rule?.timeConstraints.maxDurationSeconds}
-                              min={60}
-                            >
-                              <Hours />
-                              <Minutes />
-                              {
-                                <Text textStyle={"Body/ExtraSmall"}>
-                                  Max{" "}
-                                  {durationString(
-                                    rule?.timeConstraints.maxDurationSeconds
-                                  )}
-                                  <br />
-                                  Min 1 minute
-                                </Text>
-                              }
-                            </DurationInput>
-                          );
-                        }}
+                        render={({ field: { ref, ...rest } }) => (
+                          <DurationInput
```

Comment:
Rather than having to conditionally render the elements everywhere we use this.
What if we instead made the input elements conditionally render only if the max duration requires it

we might want to add a prop to enable or disable the default behaviour

This this example, only minutes,hours,days would be displayed, and weeks in not rendered because shouldHideUnusedInputs={true}
```
<DurationInput max={60 * 24 * 2} shouldHideUnusedInputs={true}>
<Minute/>
<Hour/>
<Day/>
<Week/>
</DurationInput>
```

Reply:
The sensible default for us may be to always hide unused inputs